fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/auto/DruidNavigator.java

```diff
@@ -217,5 +208,13 @@ private void queryDruid(SuccessCallback successCallback, String url) {
                 },
                 url
         );
+
+        try {
+            //calling get so we wait until responses are loaded before returning and processing continues
+            responseFuture.get();
+        } catch (InterruptedException | ExecutionException e) {
```

Comment:
Good point. It probably makes sense to capture it. And it's likely part of the answer for https://github.com/yahoo/fili/issues/239